### ISSUE TITLE
Data type is displayed not by Postgres

### ISSUE THREAD

**Issue by AdrianusWest** [dbeaver/dbeaver#34809]
### Description

"Table 8.1 shows all the built-in general-purpose data types. Most of the alternative names listed in the “Aliases” column are the names used internally by PostgreSQL for historical reasons. In addition, some internally used or deprecated types are available, but are not listed here."

![photo_2024-06-24_14-48-08](https://github.com/user-attachments/assets/fd96aa46-8557-4d65-9c74-cbb1ee44e2d3)

Not names, but aliases are displayed by default. Switching to display by Postgres format was not detected.

### DBeaver Version

Community Edition 24.1.2-1

### Operating System

EndeavourOS

### Database and driver

1. The Postgres database is connected.
2. The data type is displayed not by Postgres, but by the general rule.

### Steps to reproduce

1. The Postgres database is connected.
2. The data type is displayed not by Postgres, but by the general rule.

### Additional context

I emailed support with this question, but they don't support the community version, so I'm asking the question in issue format here.

Basically, the problem is simple: in Datagrip, Postgres data type mapping works by default, I didn't find support for mapping in Dbeaver Community and my colleagues didn't see it either. Is there any support for Postgres data types at all and the possibility to switch to it?

**Comment by Destrolaric** [comment]
Thank you for the idea.

**Comment by Destrolaric** [comment]
You can vote for the issue with the  emoji under the issue description to promote it to a higher priority.
